From bb9f3ecbc56dd63815a9db1cf9b0a46521fa8af8 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Thu, 29 Nov 2018 07:22:28 -0700
Subject: [PATCH 6/8] klte-common: rootdir: Stop diddling with
 /sys/power/cpufreq_*

* These aren't used anywhere in the source tree and none of the blobs
  reference them. We can drop this instead of finding/inventing an
  appropriate selinux label to allow the setattr.

Change-Id: I9e238aadade4c051c2815bdf15891443cc649624
---
 rootdir/etc/init.qcom.rc | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/rootdir/etc/init.qcom.rc b/rootdir/etc/init.qcom.rc
index 93fd5aa..67fc3a2 100644
--- a/rootdir/etc/init.qcom.rc
+++ b/rootdir/etc/init.qcom.rc
@@ -382,14 +382,6 @@ on boot
     chown system radio /sys/class/fingerprint/fingerprint/name
     chown system radio /sys/class/fingerprint/fingerprint/vendor
 
-    # SEC DVFS sysfs node
-    chown radio system /sys/power/cpufreq_max_limit
-    chown radio system /sys/power/cpufreq_min_limit
-    chown radio system /sys/power/cpufreq_table
-    chmod 664 /sys/power/cpufreq_max_limit
-    chmod 664 /sys/power/cpufreq_min_limit
-    chmod 664 /sys/power/cpufreq_table
-
     chown radio system /sys/devices/system/cpu/kernel_max
     chmod 664 /sys/devices/system/cpu/kernel_max
 
-- 
2.17.1

